io_scene_nif/modules/nif_export/property/shader/__init__.py

```diff
@@ -95,7 +95,7 @@ def export_bs_effect_shader_property(self, b_mat):
     def export_bs_lighting_shader_property(self, b_mat):
         bsshader = NifFormat.BSLightingShaderProperty()
         b_s_type = NifFormat.BSLightingShaderPropertyShaderType._enumkeys.index(b_mat.niftools_shader.bslsp_shaderobjtype)
-        bsshader.shader_type = NifFormat.BSLightingShaderPropertyShaderType._enumvalues[b_s_type]
+        bsshader.skyrim_shader_type = NifFormat.BSLightingShaderPropertyShaderType._enumvalues[b_s_type]
```

Comment:
Is the fix also required for line 138. in export_bs_shader_pp_lighting_property?
`bsshader.shader_type = NifFormat.BSShaderType._enumvalues[b_s_type]`

Reply:
I think so, as well as in the import for the same node. I just hadn't noticed it there, since I was only testing with a BSLightingShaderProperty.

Reply:
Actually, looking into nif.xml, that property seems to only exist for BSLightingShaderProperties:
![image](https://user-images.githubusercontent.com/45334438/82763391-9a5d8000-9e07-11ea-8c72-ec8a0d71c62e.png)
So it either isn't the same thing, or those lines in export_bs_shader_pp_lighting property are completely redundant.

Reply:
Actually it makes sense as this would be a F03 era block.